blockchain/blockchain-wallet-v4-frontend
# Fix/get info language

## Description
1033

## Change Type
Please enter one or more of the following: 
- Feature
- Bug Fix

## Testing Steps
Change language before registering/restoring wallet, see welcome email sent in that language.
Change language on signin page, see on login that language is updated successfully.
Change language on settings page, see that language is updated successfully.

## Code Checklist
